### PR TITLE
Added 1.15.2 support

### PR DESCRIPTION
**Pull request by yannicklamprecht**

### CODE REVIEW

File: bootstrap/pom.xml [hunk 1]
```diff
@@ -20,8 +20,8 @@
 
     <properties>
         <maven.deploy.skip>true</maven.deploy.skip>
-        <maven.compiler.source>1.6</maven.compiler.source>
-        <maven.compiler.target>1.6</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (yannicklamprecht) [inline bootstrap/pom.xml]: It is time to set "new" standards. Didn't build without it. Maven complained and recommended at least 1.7 but I think 1.8 should be best for MC.

Comment (mikroskeem) [reply bootstrap/pom.xml]: I'd split this up into separate PR.

**Comment by md-5** [comment]
https://github.com/SpigotMC/BungeeCord/pull/2729

This is gonna have to go in the readme at this point...

**Comment by yannicklamprecht** [comment]
See reason.

File: bootstrap/pom.xml (hunk 1 above)
Comment (Janmm14) [reply bootstrap/pom.xml]: That is java 6 only to warn users that they need java 8

**Pull request closed without merging**